asv_bench/benchmarks/omnisci/benchmarks.py

```diff
@@ -0,0 +1,163 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""General Modin on OmniSci backend benchmarks."""
+
+
+from ..utils import (
+    generate_dataframe,
+    RAND_LOW,
+    RAND_HIGH,
+    ASV_USE_IMPL,
+    ASV_DATASET_SIZE,
+    execute,
+)
+
+BINARY_OP_DATA_SIZE = {
+    "big": [
+        ((500_000, 20), (1_000_000, 10)),
+    ],
+    "small": [
+        ((10_000, 20), (25_000, 10)),
+    ],
+}
+
+UNARY_OP_DATA_SIZE = {
+    "big": [
+        (1_000_000, 10),
+    ],
+    "small": [
+        (10_000, 10),
+    ],
+}
+
+
+def trigger_import(dfs: list):
```

Comment:
```suggestion
def trigger_import(*dfs):
```
IMHO it's easier to just make it accept arbitrary amount of parameters for the future usage of this function